lib/auth0/api/v1/users.rb

```diff
@@ -135,13 +135,13 @@ def patch_user_metadata(user_id, metadata={})
         #
         # This will remove all your users
         def delete_users
-          path = "/api/users/"
+          path = '/api/users/'
           delete(path)
         end
 
         # {https://auth0.com/docs/api#!#delete--api-users--user_id-}
         def delete_user(user_id)
-          raise Auth0::MissingUserId, "if you want to remove all users user delete_users method" if user_id.to_s.empty?
+          fail Auth0::MissingUserId, 'if you want to remove all users user delete_users method' if user_id.to_s.empty?
```

Comment:
Yes,  an explanation in [SO](http://stackoverflow.com/questions/18811675/what-does-the-fail-keyword-do-in-ruby)